asset_dashboard/models.py

```diff
@@ -166,28 +167,47 @@ class ProjectCategory(models.Model):
     subcategory = models.TextField(null=True)
 
     def __str__(self):
-        return self.category
+        return f'{self.category} -- {self.subcategory}'
 
 
 class HouseDistrict(models.Model):
     name = models.TextField()
-    shape = models.PolygonField()
+    project = models.ForeignKey(Project,
```

Comment:
these should not  be foreign keys on house district, senate district, zone, and commissioner district.

they should be many-to-many relations on the project model.

Reply:
fixed with commit 9bc9149b0f23a29567f413cb4d3d3ac6b6ff69ec